docs/concepts/deployments.md

```diff
@@ -185,7 +185,7 @@ You may specify additional options to further customize your deployment.
 |  `-o`, `--output TEXT`            | Optional location for the YAML manifest generated as a result of the `build` step. You can version-control that file, but it's not required since the CLI can generate everything you need to define a deployment. |
 |  `-i`, `--infra`                   | The [infrastructure type](/concepts/infrastructure/) to use. (Default is `Process`) |
 |  `-ib`, `--infra-block TEXT`       | The [infrastructure block](#block-identifiers) to use, in `block-type/block-name` format. |
-|  `--override TEXT`       | One or more optional infrastructure overrides provided as a dot delimited path. For example, `env.env_key=env_value`. |
+|  `--override TEXT`       | One or more optional infrastructure overrides provided as a dot delimited path. For example, specify an environment variable: `env.env_key=env_value`. For Kubernetes, specify customizations: `customizations='[{"op": "add","path": "/spec/limits", "value": {"memory": "8Gi","cpu": "4000m"}}]'` (note the string format).|
```

Comment:
I think the path here needs review - https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
I believe a json patch for a job would be this :
`/spec/template/spec/containers/0/resources/limits`
Seems like the path is right in the tests, so perhaps just oversight?

Reply:
Nice catch, lost the middle 🍩